README.md

```diff
@@ -188,6 +189,27 @@ Short `TODO` List for new contributors:
 - Increment the test coverage.
 - [Issues marked as `Help wanted`](https://github.com/ChuckerTeam/chucker/labels/help%20wanted)
 
+### Building 🛠
+
+In order to start working on Chucker, you need to fork the project and open it in Android Studio/IntelliJ IDEA.
+
+Before committing we suggest you install the pre-commit hooks with the following command:
```

Comment:
Let's mention that the hook will be installed automatically during the first build.